docs/event-types.mdx

```diff
@@ -297,8 +297,43 @@ func main() {
 <TabItem value="rust">
 
 ```rust
-// Example TBD
-// Read the CSV and create event types
+use std::{error::Error, fs::File};
+use svix::api::{EventTypeIn, Svix};
+use tokio::runtime::Builder;
+
+async fn execute() -> Result<(), Box<dyn Error>> {
+    let svix = Svix::new(
+        "AUTH_TOKEN".to_string(),
+        None,
+    );
+
+    let mut reader = csv::ReaderBuilder::new()
+        .delimiter(b'|')
+        .from_reader(File::open("src/data.csv")?);
+
+    for result in reader.records() {
+        let record = result?;
+
+        svix.event_type()
+            .create(
+                EventTypeIn {
+                    name: record[0].to_string(),
+                    description: record[1].to_string(),
+                    ..EventTypeIn::default()
+                },
+                None,
+            )
+            .await?;
+    }
+
+    Ok(())
+}
+
+fn main() {
```

Comment:
Other folks might have a different opinion, but I think something like

```rust
#[tokio::main]
async fn main() {
    execute().await.unwrap();
}
```

Might be more idiomatic, or at least more focused on our actual lib code.

And then you can remove `use tokio::runtime::Builder;`

Reply:
Makes sense. I updated it.